api/RTCRtpSender.json

```diff
@@ -113,7 +113,7 @@
               "version_added": "67"
             },
             "edge": {
-              "version_added": null
+              "version_added": true
```

Comment:
Confirmed by https://software.hixie.ch/utilities/js/live-dom-viewer/saved/6831 in Edge 18.